Apps/Database/src/Migrations/GatewayDbContextModelSnapshot.cs

```diff
@@ -462,12 +462,10 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<DateTime>("CreatedDateTime")
                         .HasColumnType("timestamp without time zone");
 
-                    b.Property<DateTime?>("EffectiveDateTime")
-                        .IsRequired()
+                    b.Property<DateTime>("EffectiveDateTime")
```

Comment:
do we have to change EfectiveDateTime field for this story/task?

Reply:
No we don't but i don't see why not, this was automatically fixed by the dotnet ef command, most likely someone manually changed it but never created a migration.